cohort_3/curriculum/workshops/portfolio/README.md

```diff
@@ -0,0 +1,271 @@
+      pretty version. It is sheltering you!
+        - _Warning: this wording was taken from Codecademy - remember to change
+          this_
+    - We'll get our hands a bit dirtier with the basics of HTML. We'll talk a
+      little about formatting, links, and images so you can make a simple, but
+      ugly, webpage.
+        - _Warning: this wording was taken from Codecademy - remember to change
+          this_
+    - If you want to make it pretty, we'll also do a quick dive into Cascading
+      Syle Sheet (CSS), which many websites secretly use to make themselves
+      pretty.
+    - The basic building blocks of HTML are _HTML elements_. HTML elements are
+      denoted by angled brackets. These are called **tags**. `<html>`, `<head>`,
+      `<body>`, `<title>`, and `<h1>` are all tags.
+    - All of the tags listed above are referred to as start tags - and if you
+      start something, you need to end it. The end tag is exactly the same as
+      the start tag with a cute forward slash (`/`) after the `<`. Consequently
+      the end tag for `<html>` is `</html>`. The end tag for `<head>` is
+      `</head>`, and so on.
+    - HTML documents contain two important parts, denoted by tags:
+        1. **head** - contains information about the document that is _not
+           displayed on the screen_.
+        2. **body** - contains everything that id displayed on the webpage.
+    - Don't worry if you don't understand everything so far! You'll get a better
+      understanding of these concepts as you write more code. As always, the
+      most important thing is to ask a lot of questions!
+- Adding more content
+    - We previously used an `h1` tag to create a heading in our website.
+      Headings are a way to order the content in the `body`. We can use the `h1`
+      to `h6` tags to define them. `h1` is the "most important", and then these
+      tags gradually become "less important" until you reach `h6`, the "least
+      important".
+
+      ```html
+      <h1>The Onion</h1>
+      <h2>Posts</h2>
+      <h3>Son-Of-A-Gun Mouse Solves Maze Researchers Spent Months Building</h3>
+      <h3>Informal Tone Of Cover Letter Sets Job Applicant Apart From Seriously
+      Considered Candidates</h3>
+      <h2>More Heading Examples</h2>
+      <h3>A Heading</h3>
+      <h4>A Heading</h4>
+      <h5>A Heading</h5>
+      <h6>A Heading</h6>
+      ```
+      
+      ![](img/c9_heading_examples.png)
+
+    - **Challenge time:** we previously used an `h1` tag inside of the `body`
+      tag to create a heading with our name on our webpage. Create a smaller
+      heading to put the projects we create under. Here's what it should look
+      like:
+
+      ![](img/c9_create_projects_heading.png)
+
+    - Let's talk about lists. Lists can be created with either the `ul`
+      (unordered) or `ol` (ordered) list tags. We put `li` tags inside of `ul`
+      or `ol` for each element.
+
+      ```html
```

Comment:
I feel like there's a lot of info thrown at me right here. Maybe just introduce ul. They can look up ol.